schema.sql

```diff
@@ -1,5 +1,25 @@
 /* Database schema to keep the structure of entire database. */
 
+/* Created database */
+
+CREATE DATABASE vet_clinic;
+
+/* Access database */
+
+psql vet_clinic;
+
+/* Created table */
+
 CREATE TABLE animals (
     name varchar(100)
 );
```

Comment:
- [ ] Currently, there are two tables named `animals` in your schema.sql file. Kindly remove this one so we can make use of the one that has all the data.

Reply:
Done